src/main/java/com/google/codeu/servlets/CountryServlet.java

```diff
@@ -45,14 +45,33 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
 
     UserService userService = UserServiceFactory.getUserService();
+    
     String requestUrl = request.getRequestURI();
     String countryCode = requestUrl.substring("/country/".length());
+    String category = null;
 
     // Confirm valid url format
     if (countryCode == null || countryCode.equals("")) {
       response.sendRedirect("/");
       return;
     }
+    
+    // Redirect to categories servlet if requesting category
+    CharSequence catagoryURL = "/c";
+    System.err.println(requestUrl);
+    if (countryCode.contains(catagoryURL)) {
+      System.err.println("Has /c");
```

Comment:
I keep forgetting to take these out, too, but make sure you take out debugging things before merging.

Reply:
My bad! Thank you for catching them!